website/src/app/documentation/demos/daterangepicker/daterangepicker.demo.html

```diff
@@ -143,6 +164,23 @@ <h3><code class="clr-code">clrDaterangepicker</code> directive</h3>
       </form>
       <clr-code-snippet [clrCode]="basicDemo"></clr-code-snippet>
 
+      <h3>TimeSelection</h3>
+
+      <p>
+        To use the daterangepicker, add the <code class="clr-code">clrDaterangepicker</code> directive to an
```

Comment:
maybe change the text to "To use the daterangepicker with time selection, set timeSelection=true additionally. The value will be..."